### ISSUE TITLE
Any details about the stage1 pretrain?

### ISSUE THREAD

**Issue by gd2016229035** [Vision-CAIR/MiniGPT-4#428]
Great job! 
When I use your publicly available stage3 finetuning code, I can achieve similar accuracy. 
However, when I try to train stage1 and stage2 on my own, there is a significant accuracy difference, especially for the refcoco task. I have a couple of questions: 
(1) Does the caption dataset used in stage1 need to be preprocessed? What methods and how much data is approximately used? (2) Does the grit dataset need cleaning? When constructing the GRIT REC task, does grit choose the "ref_exps" field? How do you select training samples from multiple ref_exps boxes, picking the one with the highest confidence? 
Looking forward to your reply!

**Comment by XenonLamb** [comment]
I also observed similar results to yours... I tried to start from the MiniGPTV1's stage1 and 2 by adding the Caption/REC/REG/VQA datasets mentioned in the v2 paper (except for GRIT where I don't know how the groundtruth is utilized). I performance on VizWiz and HatefulMemes got improved but there's notable degrade from v1's stage1+stage2 + v2's stage 3  on OKVQA and GQA (even though these 2 tasks are in the training datasets).
